components/AppShell.vue

```diff
@@ -6,128 +6,64 @@
-      <div class="flex w-full justify-between md:hidden">
-        <!-- FIXME: This should be a button -->
-        <div
-          type="button"
-          class="flex items-center pl-6 focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
-          @click="sidebarOpen = true"
-        >
-          <div>
-            <span class="sr-only">Open sidebar</span>
-            <navigation-menu class="text-content-tertiary" />
-          </div>
-        </div>
-        <NuxtLink
-          to="/"
-          class="flex items-center justify-center h-16 w-16 bg-brand-primary focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
-        >
-          <centrapay-logo class="text-content-on-color icon-2xl" />
-        </NuxtLink>
-      </div>
-      <div class="hidden md:flex md:flex-1 md:justify-between md:py-3">
+      <div class="flex w-full justify-between">
         <div class="flex items-center">
           <NuxtLink
             to="/"
             class="flex items-center justify-center h-16 w-16 bg-brand-primary focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
+            @click="sidebarOpen = false"
           >
             <centrapay-logo class="text-content-on-color icon-2xl" />
           </NuxtLink>
           <div class="flex flex-row space-x-1 ml-7">
             <NuxtLink
-              v-for="item in primaryNavigation"
-              :key="item.name"
-              :href="item.href"
-              :target="item.target"
-              class="text-gray-600 text-sm leading-5 font-medium px-3 py-2 rounded-lg hover:bg-gray-200 hover:text-content-primary focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
-              :class="item.current ? 'bg-gray-100': ''"
+              href="/"
+              target="_self"
+              class="text-gray-600 text-sm leading-5 font-medium px-3 py-2 rounded-lg bg-gray-100 hover:bg-gray-200 hover:text-content-primary focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
+              @click="sidebarOpen = false"
             >
-              {{ item.name }}
+              Docs
             </NuxtLink>
           </div>
         </div>
+        <button
+          class="flex items-center pr-6 md:hidden focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
+          @click="sidebarOpen = !sidebarOpen"
+        >
+          <span class="sr-only">Open sidebar</span>
+          <navigation-menu v-if="!sidebarOpen" class="block h-6 w-6 text-content-tertiary" aria-hidden="true" />
+          <close-outline v-else class="block h-6 w-6 text-content-tertiary" aria-hidden="true" />
+      </button>
       </div>
     </div>
 
     <!-- Static sidebar for desktop -->
-    <div class="hidden border-t md:fixed md:h-full md:inset-y-16 md:flex md:w-64 xl:w-80 md:flex-col">
+    <div class="hidden border-t md:fixed md:h-full md:inset-y-16 md:flex md:pt-5 md:w-64 xl:w-80 md:flex-col">
```

Comment:
Added a little padding between the desktop sidebar and the nav bar (header)

Before:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/46909693/214225906-37433a3c-cb24-4c16-a954-e0865db32ddd.png">

After:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/46909693/214225839-ba8ef371-80c7-49a7-823b-b2cbda62b012.png">

Reply:
@tom-blake I think this should go in the div below with the border as there is a gap on the RHS border

Reply:
@MeganSteenkamp Great spot - updated
![image](https://user-images.githubusercontent.com/46909693/214437753-8a6afc29-e6da-4a77-82ac-d35dc7a99fa9.png)